File: aula-um/retangulo.cpp
#include <stdio.h>

class Retangulo
{
public:
    float largura, altura;

    float area()
    {
        return this->altura * this->largura;
    }

    float perimetro()
    {
        return (2 * this->largura) + (2 * this->altura);
    }
};

int main()
{
    Retangulo r;

    printf("Digite a largura em cm: ");
    scanf("%f", &r.largura);
    printf("Digite a altura em cm: ");
    scanf("%f", &r.altura);

    printf("Area: %.2f cm quadrados\n", r.area());
    printf("Perimetro: %.2f cm\n", r.perimetro());

    return 0;
}

File: 3-poo-com-java/Retangulo/Retangulo.java
public class Retangulo {
    private int altura;
    private int largura;

    public Retangulo(int altura, int largura) {
        this.altura = altura;
        this.largura = largura;
    }

    public int area() {
        return this.altura * this.largura;
    }

    public int perimetro() {
        return (2 * this.altura) + (2 * this.largura);
    }

    public int getAltura() {
        return this.altura;
    }

    public int getLargura() {
        return this.largura;
    }
}
